crates/zkvm/adapters/risc0/src/host.rs

```diff
@@ -197,8 +185,9 @@ outputs",
 
         let filename = "proof-groth16.bin";
         let mut file = File::create(filename).unwrap();
-        file.write_all(proof.as_bytes()).unwrap();
+        file.write_all(&bincode::serialize(&proof).expect("bincode serialization failed"))
+            .unwrap();
```

Comment:
Should probably be a little better about this.  I didn't catch the `.unwrap()` before.  Also *must* this be bincode here or can it be something more generic/stable?

Reply:
It need not be `bincode`. 
The `.unwrap()` is here used in the test to save the proof locally since generating it takes some time.

Reply:
But since the proof is already serialized, we can skip this serialization. Thanks!

Reply:
Oh my bad.